subprojects/gradle/slack-test-fixtures/src/main/kotlin/com/avito/slack/FakeSlackClient.kt

```diff
@@ -57,4 +58,8 @@ class FakeSlackClient : SlackClient {
         updatedMessageTimestamp = messageTimestamp
         return Try.Success(SlackMessage.createStubInstance())
     }
+
+    override fun uploadHtml(channel: SlackChannel, message: String, file: File): Try<Unit> {
+        TODO("Not yet implemented")
```

Comment:
Comment please about plans.
Without an Html report, I'd like to see a link to it or to a build at least.

Reply:
i just didn't come up with any tests using this function for now